packages/server/src/api/routes/tests/row.spec.ts

```diff
@@ -102,7 +103,7 @@ describe.each([
   ): SaveTableRequest {
     const defaultSchema: TableSchema = {
       id: {
-        type: FieldType.AUTO,
```

Comment:
The `id` field is mandatory for external ds, and having it as auto causes conflicts with parallel creations. Having them as `number` solves that issue and its closer to the real BB use cases

Reply:
This is correct anyway - no columns actually have a `FieldType.AUTO` type.